src/main/java/com/home_work/authentication/TableServlet.java

```diff
@@ -0,0 +1,86 @@
+package com.home_work.authentication;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebInitParam;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.*;
+
+/*
+GET /table/{tableName} - return content of file/table
+DELETE /table/{tableName} - deletes file
+PUT /table/{tableName} - update existing file
+POST /table/{tableName} - creates new file
+ */
+@WebServlet(name = "tableServlet", value = "/tables/*")
+public class TableServlet extends HttpServlet {
+
+    String direction = "/home/dana/Documents/projects/hw/web/src/main/java/com/home_work/authentication/db";
```

Comment:
Let's try to extract logic related to files to a separate class to avoid duplication of direction variables.